tests/dtypes/test_external_ref.py

```diff
@@ -75,3 +75,51 @@ def test_external_ref(
             'country.code': 'lt'
         }
     ]
+
+
+def test_external_ref_without_primary_key(
+    rc: RawConfig,
+    postgresql: str,
+    request: FixtureRequest,
+):
+    context = bootstrap_manifest(rc, '''
+    d | r | b | m | property | type   | ref                           | source       | level | access
+    datasets/externalref     |        |                               |              |       |
+      | external             | sql    |                               | sqlite://    |       |
+      |   |   | Country      |        |                               |              |       |
+      |   |   |   | code     | string |                               |              |       | open
+      |   |   |   | name     | string |                               |              |       | open
+    datasets/internal/pk     |        |                               |              |       |
+      |   |   | City         |        |                               |              |       |
+      |   |   |   | name     | string |                               |              |       | open
+      |   |   |   | country  | ref    | /datasets/externalref/Country |              | 3     | open
```

Comment:
Please add tests with `ref: Country[id]`. See my testing notes, with `CityExplicit`. If referenced model does not have `model.ref` set, but referenced column is given in `[]`, then we should used referenced columns in `[]`.